src/utils.js

```diff
@@ -268,6 +271,37 @@ export function getIsbnIssnTemplate(queryTemplate, identifierTypes, queryIndex)
   return template(queryTemplate)({ identifierTypeId });
 }
 
+const getAdvancedSearchQueryTemplate = (queryIndex, matchOption) => fieldSearchConfigurations[queryIndex]?.[matchOption];
+
+export function buildAdvancedSearchQuery(queryParams, pathComponents, resourceData, logger, props) {
```

Comment:
Can we utilize the `buildQuery` function from the `builtManifestObject.js` file? By simply adding an `if (queryIndex === AdvancedSearch)` condition?